sstables/generation_type.hh

```diff
@@ -35,7 +35,11 @@ public:
     constexpr int_t value() const noexcept { return _value; }
 
     explicit operator data_value() const noexcept {
-        return data_value(_value);
+        return data_value(utils::UUID(1 << 12, _value));
```

Comment:
Why this specific magic number?
Can it be defined in logical terms as some function of something? and be used below (instead of a seemingly arbitrary contstant)

Reply:
i replaced it with a const variable for better readability.

Reply:
Isn't there a function in UUID_gen to make a time uuid from seconds/milliseconds?

Reply:
ahh, true. i missed that. now switched to the `UUID_gen`. thanks for the reminder.